lib/ansible/modules/windows/win_robocopy.ps1

```diff
@@ -92,74 +93,61 @@ $result.flags = $flags
 
 $robocopy_output = ""
 $rc = 0
-If ($check_mode -eq $true) {
-    $robocopy_output = "Would have copied the contents of $src to $dest"
-    $rc = 0
-}
-Else {
-    Try {
-        &robocopy $robocopy_opts | Tee-Object -Variable robocopy_output | Out-Null
-        $rc = $LASTEXITCODE
-    }
-    Catch {
-        $ErrorMessage = $_.Exception.Message
-        Fail-Json $result "Error synchronizing $src to $dest! Msg: $ErrorMessage"
-    }
+Try {
+
+    &robocopy $robocopy_opts | Tee-Object -Variable robocopy_output | Out-Null
```

Comment:
This line is bothering me.  
$robocopy_output = &robocopy $robocopy_opts
would seem to be simpler.  Why use Tee-Object to make 2 copies of the output and then throw the pipeline output away?